backend/apps/blogs/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.conf import settings
+
+
+class BlogPost(models.Model):
+
+    title = models.CharField(max_length=150)
+    text = models.TextField()
+    publishDate = models.DateTimeField(auto_now_add=True)
```

Comment:
Today I learned that `auto_now_add` exists, that's very useful 🤔

Should be snake_case
```suggestion
    publish_date = models.DateTimeField(auto_now_add=True)
```